agency/test/hooks/issuer-profile.test.ts

```diff
@@ -0,0 +1,53 @@
+import { HookContext } from '@feathersjs/feathers';
+import { checkValidIssuerProfile } from '../../src/utils/hooks/issuer-profile';
+import memory from 'feathers-memory';
+import app from '../../src/app';
+import { DuplicatedProfileError } from '../../src/models/errors';
+import { GeneralError } from '@feathersjs/errors';
+import profile from '../data/profile.json';
```

Comment:
`profile.json` does not seem to be committed

Reply:
Should be committed now